recipes/r-hdrcde/meta.yaml

```diff
@@ -1,94 +1,49 @@
+{% set version = '3.1' %}
```

Comment:
Can you use this style for all the new recipes in this PR please? It is the new default for r packages that I introduced in conda-build 2.1.9.